File: Chicken Sexing.js
function correctness(bobsDecisions, expertDecisions) {
  let result = 0
  bobsDecisions.map((v,i) =>  {
      if(bobsDecisions[i]==expertDecisions[i]){
         result+=1
      }else if(bobsDecisions[i]!=expertDecisions[i] && (bobsDecisions[i]=="?" || expertDecisions[i]=="?")){
        result+=0.5
      }else{
        result+=0
      }
  })
  return result
}

File: Chicken Sexing/Chicken Sexing.py
def correctness(a, b): 
    result = 0
    for i in range(len(a)):
        if a[i]==b[i]:
            result+=1
        elif a[i]!=b[i] and (a[i]=="?" or b[i]=="?"):
            result+=0.5
    return result
